fedmsg_meta_fedora_infrastructure/conglomerators/tagger/tags.py

```diff
@@ -0,0 +1,54 @@
+import fedmsg.meta.base
+from fedmsg_meta_fedora_infrastructure.fasshim import avatar_url
+
+class AbstractTaggerConglomerator(fedmsg.meta.base.BaseConglomerator):
+
+    def merge(self, constituents, **config):
+        ms = constituents  # shorthand
+
+        user = constituents[0]['msg']['user']['username']
```

Comment:
Here we use ``constituents``, below we use ``ms``, we're not being very consistent here